internal/testrunner/runners/pipeline/runner.go

```diff
@@ -248,6 +250,19 @@ func (r *runner) verifyResults(testCaseFile string, config *testConfig, result *
 	return nil
 }
 
+// stripEmptyTestResults function removes events which are empty strings. These strings can represent
+// documents processed by a pipeline which potentially used a "drop" processor (to drop the event at all).
+func stripEmptyTestResults(result *testResult) *testResult {
```

Comment:
Maybe add a unit test for this function?

Reply:
Added